NHLBI_BioData_Catalyst/R/4_Genomic_Queries.ipynb

```diff
@@ -134,8 +138,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "library(stringr)\n",
-    "library(dplyr)"
+    "#library(stringr)\n",
```

Comment:
Needed to uncomment these to have the notebook run successfully in Terra.

Reply:
Resolved in [e421aff3e104214b34879bf5c4551b4d612f44c4](https://github.com/hms-dbmi/Access-to-Data-using-PIC-SURE-API/commit/e421aff3e104214b34879bf5c4551b4d612f44c4)